deeplake/core/dataset/deeplake_query_dataset.py

```diff
@@ -331,5 +332,10 @@ def __del__(self):
     def random_split(self, lengths: Sequence[Union[int, float]]):
         if math.isclose(sum(lengths), 1) and sum(lengths) <= 1:
             lengths = calculate_absolute_lengths(lengths, len(self))
-        vs = self.indra_ds.random_split(lengths)
+        try:
+            seed = np.random.get_state()[1][0]
```

Comment:
just want to double check on seed choice of numpy vs random? we would need to ensure consistency across deeplake randomness.